Core/Services/UserService.cs

```diff
@@ -57,5 +57,18 @@ public async Task<string> GetUserRoleAsync(User user)
 
             return userRoles.First(); // we get only the first role, because the user will have only one
         }
+
+        public async Task<UserEditProfileInfoDTO>  UserEditProfileInfo(UserEditProfileInfoDTO userEditProfileInfo, string userId)
+        {
+            var updateUser = await _userManager.FindByIdAsync(userId);
+
+            updateUser.Name = userEditProfileInfo.Name;
+            updateUser.Surname = userEditProfileInfo.Surname;
+            updateUser.Email = userEditProfileInfo.Email;
```

Comment:
also we should change [UserName], [NormalizedUserName], [NormalizedEmail] and set field [EmailConfirmed] - false